hunts/src/puzzlesSlice.js

```diff
@@ -243,10 +244,23 @@ export const selectAllTags = createSelector(
           : tagNames.add(tag.name) && [...uniqueTags, tag],
       []
     );
-    uniqueTags.sort(
+
+    const defaultTagNames = DEFAULT_TAGS.map((tag) => tag.name);
+    const defaultTags = uniqueTags.filter((tag) =>
+      defaultTagNames.includes(tag.name)
+    );
```

Comment:
Isn't defaultTags the same as just DEFAULT_TAGS?

Reply:
No, as mentioned, defaultTags contains the versions of the objects backed by the DB, but DEFAULT_TAGS are just the constants that may not exist in the DB yet. 

I think the root of this jankiness is the fact we're defining default tags outside of the DB, but it would require a bit more work to do it the proper way.

Reply:
Oh, I see. Can we add a comment here about this, too? This DB vs. constants tags is kinda subtle.

It'd be nice to create an issue for just storing the tags in the DB and then put a `// TODO(#<ISSUE_NUM>)` comment here.

Reply:
Created #527, will add it to the TODO in constants.js that is about this.